lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -709,7 +709,7 @@ export abstract class CacheManager implements ICacheManager {
     async removeAccount(accountKey: string): Promise<void> {
         const account = this.getAccount(accountKey);
         if (!account) {
-            throw ClientAuthError.createNoAccountFoundError();
+            return;
```

Comment:
Does the end user ever call this? Swallowing the error is okay, should have been there from the beginning as it should be the "we will do best to remove but do not fail in case the account isn't".

Just curious if there is any end user impact.

Reply:
Not directly AFAIK, always through a logout API. Even if it is though, this shouldn't impact anything. The end result we're looking for here is that the account is not cached when the promise resolves, the state of the cache before the function is called is irrelevant.